### DISCUSSION TITLE
Баги BiglyBT 3.3.0.0

### DISCUSSION THREAD

**Discussion by naumgod** [BiglySoftware/BiglyBT#2901] [Welcome]
баги BiglyBT 3.3.0.0.

1. При скачивании фала .torrent  в браузере хром. Нажимаю открыть И баг не выпрыгивает на передний экран открытие файла куда сохранять.
2. Второе что НЕ очевидно и не удобно куда сохранять файлы торрента. Замаскированно и не понятно.
3. Нет выбрать минималистический вид. Очень много лишнего везде что бесит и мешает.
4. Нет возможности в колонках добавить столбец Сиды и пиры! А не где то там в низу в дебрях искать этих пиров. 

Это всё не даёт нормально и с наслаждением использовать программу! Всё не удобно и поэтому не пользуется спросом! 
Сделайте свой продукт удобным!

**Comment by parg** [comment]
Google translation:

BiglyBT 3.3.0.0 bugs.

When downloading a .torrent file in the chrome browser. I press open And the bug does not jump out to the front screen opening the file where to save it.

The second thing is NOT obvious and not convenient where to save torrent files. Disguised and incomprehensible.

No choose a minimalist look. A lot of superfluous everywhere that infuriates and interferes.

There is no way to add Seeds and peers column in columns! And not somewhere down there in the wilds to look for these feasts.

All this does not allow you to use the program normally and with pleasure! Everything is not convenient and therefore not in demand!
Make your product convenient!

**Comment by parg** [comment]
Sounds like you need a different, convenient bittorrent client, good luck.

**Comment by naumgod** [comment]
Sounds like you don't want to make a program user-friendly. It is not the strongest who survives, but the most adaptable. You don't want to understand your mistakes. And you're not morals! It's nice to stay!

**Comment by parg** [comment]
Whatever
